wiki/Beatmap/Break/fr.md

```diff
@@ -7,10 +7,14 @@ tags:
   - temps de pause
   - repos
   - temps de repos
+outdated_since: 2070de1a812bbf985e696a679d917f61edf5bd67
+outdated_translation: true
 ---
 
 # Pause
 
 Les **pauses** sont des sections d'une [beatmap](/wiki/Beatmap) qui n'ont aucun [objet](/wiki/Gameplay/Hit_object). Pendant ce temps, le [drain de vie](/wiki/Gameplay/Health) est temporairement interrompu. Les pauses permettent généralement au joueur de reposer son bras pendant un court instant et de repositionner son [dispositif d'entrée](/wiki/Gameplay/Input_device).
 
 Selon les [paramètres](/wiki/Client/Beatmap_editor/Song_setup) activés, des letterboxes peuvent apparaître lorsqu'une pause est initiée. En outre, un graphique et un son de réussite ou d'échec peuvent apparaître pour indiquer si le joueur s'en sort bien ou non. Cela n'apparaîtra que s'il y a suffisamment de temps pour que l'animation s'affiche.
+
+![Capture d'écran d'une passe de section de pause](img/Section_pass.jpg "Passe de section")
```

Comment:
```suggestion
![Capture d'écran d'une section réussi sur l'écran de pause](img/Section_pass.jpg "Section réussi") ![Capture d'écran d'un section miss sur l'écran de pause](img/Section_fail.jpg "Section miss")
```

Reply:
yeah that was intentional, I was sort of on a time limit.